1.  **Overview of various commonly used PLINK files**

    .. image:: plink_files.jpg

2.  **Working example of three single nucleotide polymorphisms (SNPs) aggregated into a single individual polygenic risk score (PRS).**
    *The weight is either the beta or the log of the odds‐ ratio, depending on whether a continuous or binary trait is analysed

    .. image:: PRS.jpg

3.  **The PRS analysis process**

    .. image:: GWAS_PRS.png

4.  **Shown is a flow chart of suggested analytical steps that can be followed to perform QC and select software for PRS analyses**

    .. image:: PRS_software.png


https://github.com/MareesAT/GWA_tutorial/ <https://github.com/MareesAT/GWA_tutorial/>`_

`Marees A T, de Kluiver H, Stringer S, et al. A tutorial on conducting genome‐wide association studies: Quality control and statistical analysis[J]. International journal of methods in psychiatric research, 2018, 27(2): e1608. <https://onlinelibrary.wiley.com/doi/full/10.1002/mpr.1608>`_

`https://gwaslab.org/ <https://gwaslab.org/>`_

**All association analyses were performed using REGENIE**,excluded variants with Hardy-Weinberg equilibrium (HWE) test P > 1 x 10-15 and >10% missingness.

`Gaynor S M, Joseph T, Bai X, et al. Yield of genetic association signals from genomes, exomes, and imputation in the UK biobank[J]. medRxiv, 2023: 2023.09. 13.23295479. <https://www.medrxiv.org/content/10.1101/2023.09.13.23295479v1>`_

`REGENIE (https://rgcgithub.github.io/regenie/) <https://rgcgithub.github.io/regenie/>`_

`Mbatchou, J., Barnard, L., Backman, J. et al. Computationally efficient whole-genome regression for quantitative and binary traits. Nat Genet 53, 1097–1103 (2021). <https://doi.org/10.1038/s41588-021-00870-7>`_

`Bycroft C, Freeman C, Petkova D, et al. The UK Biobank resource with deep phenotyping and genomic data[J]. Nature, 2018, 562(7726): 203-209. <https://www.nature.com/articles/s41586-018-0579-z>`_

`GWAS Catalog: https://www.ebi.ac.uk/gwas/ <https://www.ebi.ac.uk/gwas/>`_

`GCTA a tool for Genome-wide Complex Trait Analysis <https://yanglab.westlake.edu.cn/software/gcta/#Overview>`_

`Gurinovich A, Li M, Leshchyk A, et al. Evaluation of GENESIS, SAIGE, REGENIE and fastGWA-GLMM for genome-wide association studies of binary traits in correlated data[J]. Frontiers in Genetics, 2022, 13: 897210. <https://www.frontiersin.org/articles/10.3389/fgene.2022.897210/full>`_

`https://choishingwan.github.io/PRS-Tutorial/ <https://choishingwan.github.io/PRS-Tutorial/>`_

`https://2cjenn.github.io/PRS_Pipeline/ <https://2cjenn.github.io/PRS_Pipeline/>`_

`PRSICE2:https://choishingwan.github.io/PRSice/ <https://choishingwan.github.io/PRSice/>`_

`Collister J A, Liu X, Clifton L. Calculating polygenic risk scores (PRS) in UK Biobank: a practical guide for epidemiologists[J]. Frontiers in genetics, 2022, 13: 818574. <https://www.frontiersin.org/articles/10.3389/fgene.2022.818574/full>`_

`Choi S W, Mak T S H, O’Reilly P F. Tutorial: a guide to performing polygenic risk score analyses[J]. Nature protocols, 2020, 15(9): 2759-2772. <https://www.nature.com/articles/s41596-020-0353-1>`_
